docker/Dockerfile

```diff
@@ -33,13 +33,20 @@ RUN \
     rm --force --recursive --verbose "${ARTIFACT_DIR}"
 
 ENV \
-    RUN_USER="${RUN_USER:-www-data}" \
+    BASH_ENV="/etc/profile.d/remap_envvars.sh" \
+    FILE_UPLOAD_DIR="${PERSISTENCE_EXCLUSIONS_DIR:-/var/lib/portal/uploads}" \
```

Comment:
typo - didn't you mean:
`FILE_UPLOAD_DIR="${FILE_UPLOAD_DIR:-/var/lib/portal/uploads}"` ?